.travis.yml

```diff
@@ -37,32 +37,36 @@ rvm:
 matrix:
   fast_finish: true
   include:
-    - rvm: 2.2 
-      dist: trusty 
+    - rvm: 2.2.10
```

Comment:
In the future @MSP-Greg, I'd prefer you make changes to travis.yml in separate PRs. That way, they won't get blocked by the corresponding feature/bugfix.

Reply:
I was thinking the same.

While I was testing/debugging, Travis seemed to be acting up, and I wasn't sure if it was affecting the CI results I was seeing.. IOW, didn't just decide to throw it in, I was having failures...

How about I remove it right now & add another PR?